rootfs/scheduler/__init__.py

```diff
@@ -936,62 +936,6 @@ def delete_rc(self, namespace, name):
 
         return response
 
-    def _healthcheck(self, namespace, container, routable=False, path='/', port=5000,
```

Comment:
so you want to move the k8s healthcheck schema over to the `scheduler` module instead of sticking it in the serializer? That makes sense. Just trying to grok what you wanna change :)